docs/getting-started/GCE.md

```diff
@@ -63,7 +63,9 @@ gcloud compute instances create \
   --metadata-from-file user-data=<PATH_TO_CLOUD_CONFIG>/user-data-first
 ```
 
-For the second server run:
+Open your `user-data-others` file and replace the instances of `172.17.8.101` with this private IP address of the `calico-01` server you just created.
```

Comment:
with this -> with the

Also, we don't tell the user how to determine the private IP address do we?  We explicitly tell them for AWS and DigitalOcean.